databases/week2/school_database.sql

```diff
@@ -0,0 +1,59 @@
+-- DROP DATABASE IF EXISTS `lesson2_homework_school_database`;
+-- SHOW DATABASES;
+
+-- Create a new database containing the following tables:
+CREATE DATABASE IF NOT EXISTS `lesson2_homework_school_database`;
+
+USE `lesson2_homework_school_database`;
+
+-- Class: with the columns: id, name, begins (date), ends (date)
+CREATE TABLE IF NOT EXISTS `class` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `begins` DATETIME NOT NULL,
+  `ends` DATETIME NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+CREATE TABLE `student` (
+  `id` int(3) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `email` varchar(255) NOT NULL,
+  `phone` varchar(255) NULL,
```

Comment:
Is it ideal to assign 255 characters for human name, email or phone..? Typically max charfor a phone would not be more than 15. Please discover why & how we assign & what is the advantages of assigning an appropriate size(char length)

Reply:
Made changes:
`name` varchar(60) NOT NULL,
  `email` varchar(60) NOT NULL,
  `phone` varchar(15)